### DISCUSSION TITLE
Footnotes and multiple references

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#10976] [Q&A] [answered]
### Description

For Footnotes I have 3 annotations referring to the same reference. I want have it listed just once but I code it as below and when I save  Quarto will convert each [^mac] to be {^1], [^2], [^3} and make a correlating footnote for each. I want it to just be ONE footnote for  the 3 refs.

Thanks

````qmd
---
title: "Making FootNotes"

---

## Noted

Ref one. [^mac] Ref 2. [^mac] Ref 3. [^mac}

[^mac]: MacCarthy, James, et al. “The Latest Data Confirms: Forest Fires Are Getting Worse.” World Resources Institute, August 13, 2024.



The end.
````

Output: 
![image](https://github.com/user-attachments/assets/34280ae8-0b31-42b6-9bc5-031361325933)

**Comment by mcanouil** [comment, marked as answer]
Please search in issues and discussions before opening a new thread.

- https://github.com/quarto-dev/quarto-cli/issues/5440

**Comment by IceManTBM3** [comment]
I thought I had. That said, as I read this, this is a known Pandoc issue and there is no workaround at this time?

Thanks

**Reply by cscheid** [reply]
That's correct.
